invenio/b2share/modules/b2deposit/b2share_model/metadata/linguistics_lang_codes.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 
 lang_codes = [
+    ('eng', 'English'),
```

Comment:
@emanueldima why did you duplicate the `eng` row? You should not modify the closed vocabulary. If you intend to set a default value then this should be done modifying `HTML5ModelConverter.py:180` to support that, as the order or further processing of the list might be added/done. But if not, then at least remove the duplicate `eng` row that appears later. There is a new 2015 revision of ISO 639-3 out, and when rebuilding the pair-list this might easily be overseen.